### PR TITLE
Adding tests for python 3.7

### PR DESCRIPTION
**Pull request by iitzco-asapp**
- `tox` now must execute test suite in `3.7`
- CCI was fixed so that it can install requirements in `3.7`

**The python:3.6 image we are using already contains both python 3.6 and python 3.7**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -28,6 +32,10 @@ jobs:
         - image: python:3.6
```
Comment (jeremyasapp) [inline .circleci/config.yml]: isn't this a problem?

Comment (iitzco-asapp) [reply .circleci/config.yml]: This image also provides `3.7`

**Comment by iitzco-asapp** [comment]
> Just wanted to clarify - this will have python 3.7 used to run our tests only? I realize the image has both installed, but how are we testing / what are we officially supporting?

`tox` FTW!

**Comment by iitzco-asapp** [comment]
> There is a lot of overlap in the config.yml besides what you put in the anchor. Was there a reason why you limited it to just those 2 python 3 install steps?

`python3` anchor is just to install python3.7. It is not needed when publishing docs or the package. Naming is not the best I think... I will update.

**Pull request merged**